.github/workflows/nightly.yml

```diff
@@ -0,0 +1,16 @@
+name: Nightly
+
+on:
+    schedule:
+        - cron: "0 0 * * *"
+    workflow_dispatch:
+
+jobs:
+    Create-Tasks-Summary:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: dev/raging-echos
```

Comment:
Hardcoded on the dev branch for now - can remove this when that's merged to `main`